changelogs/fragments/1216-wafv2_web_acl-return.yml

```diff
@@ -0,0 +1,5 @@
+bugfixes:
+- wafv2_web_acl - consistently return web ACL info as described in module documentation (https://github.com/ansible-collections/community.aws/pull/1216).
+- wafv2_web_acl - fix ``changed`` status when description not specified (https://github.com/ansible-collections/community.aws/pull/1216).
+minor_changes:
+- wafv2_web_acl - relax botocore requirement to bare minimum required (https://github.com/ansible-collections/community.aws/pull/1216).
```

Comment:
looks like this got added from your other PR :)

Reply:
Yup, you'll see two commits, unfortunately GitHub doesn't support chaining PRs in the same way it's possible to chain them in Gerrit.